homeassistant/components/unifiprotect/strings.json

```diff
@@ -75,10 +75,6 @@
     "ea_setup_failed": {
       "title": "Setup error using Early Access version",
       "description": "You are using v{version} of UniFi Protect which is an Early Access version. An unrecoverable error occurred while trying to load the integration. Please [downgrade to a stable version](https://www.home-assistant.io/integrations/unifiprotect#downgrading-unifi-protect) of UniFi Protect to continue using the integration.\n\nError: {error}"
-    },
-    "deprecate_smart_sensor": {
```

Comment:
This is referenced in test `test_deprecate_smart_default`

Reply:
Thanks, I removed the tests.